lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -189,6 +189,25 @@ function _signup_resource_index($user, $users, $campaigns, $count, $page, $compe
     'runs' => $runs,
   ];
 
+  if (variable_get('rogue_collection', FALSE)) {
+    if ($user) {
+      $northstar_user_id = dosomething_user_get_northstar_id($user->uid);
+    }
+
+     $rogue = dosomething_rogue_client();
+
+     $response = $rogue->getActivity([
+       'filter' => [
+         'northstar_id' => $northstar_user_id,
+         'campaign_id' => $campaigns[0],
+       ],
+     ]);
+
+     if ($response) {
+      return $response;
```

Comment:
that's what I was thinking! However, when I try to return it, it just returns an empty data set and seems to be hitting the `return (new SignupTransformer)->index($parameters);` line (when I die dump right before that line, it appears there :-/) any idea what I'm doing wrong?